pyanaconda/dbus/launcher.py

```diff
@@ -130,3 +148,53 @@ def stop(self):
             log.error("DBus daemon can't be killed!")
         elif ret_code != 0:
             log.error("DBus daemon exited with error %s", ret_code)
+
+    def _set_environment(self):
+        """Set the environment variables."""
+        # pylint: disable=environment-modify
+        os.environ[DBUS_ANACONDA_SESSION_ADDRESS] = self._bus_address
+
+    def _write_bus_address(self):
+        """Write the bus address to a file."""
+        file_name = ANACONDA_BUS_ADDR_FILE
+        run_dir = os.path.dirname(file_name)
+
+        if not os.path.exists(run_dir):
+            os.mkdir(run_dir)
+
+        with open(file_name, 'wt') as f:
+            f.write(self._bus_address)
+
+    def _remove_bus_address(self):
```

Comment:
I would rename this to `_remove_bus_address_file` to be more explicit.

Reply:
Fixed.